debug_toolbar/panels/sql/tracking.py

```diff
@@ -87,10 +100,10 @@ class NormalCursorWrapper(BaseCursorWrapper):
     Wraps a cursor and logs queries.
     """
 
-    def __init__(self, cursor, db, logger):
+    def __init__(self, cursor, connection, logger):
         self.cursor = cursor
         # Instance of a BaseDatabaseWrapper subclass
-        self.db = db
+        self.connection = connection
```

Comment:
Please keep the name `db` rather than `connection`. It should match the [cursor signature for Django](https://github.com/django/django/blob/main/django/db/backends/utils.py#L15)

Reply:
Sounds good.  Dropped that entire renaming commit.